Dockerfile

```diff
@@ -1,33 +1,17 @@
-FROM debian:buster-slim
-
-# Debian Base to use
-ENV DEBIAN_VERSION buster
-
-# initial install of av daemon
-RUN echo "deb http://http.debian.net/debian/ $DEBIAN_VERSION main contrib non-free" > /etc/apt/sources.list && \
-    echo "deb http://http.debian.net/debian/ $DEBIAN_VERSION-updates main contrib non-free" >> /etc/apt/sources.list && \
-    echo "deb http://security.debian.org/ $DEBIAN_VERSION/updates main contrib non-free" >> /etc/apt/sources.list && \
-    apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install ruby-full -y && \
+FROM clamav/clamav:stable
+
+UsER root
```

Comment:
```suggestion
USER root
```

Reply:
weird though this is either doing nothing or Dockerfiles are smart enough to manage with a case typo